content/kubernetes/deployment/deployment-options.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: Flexible deployment options
+linktitle: Deployment options
+description: Redis Enterprise for Kubernetes allows you to deploy to multiple namespaces. This article describes flexible deployment options you can use to meet your specific needs. 
+weight: 12
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/deployment/deployment-options.md,
+    /kubernetes/deployment/deployment-options/,
+]
+
+---
+Redis Enterprise for Kubernetes can be deployed in several different ways depending on your database needs.
+
+A single Redis Enterprise cluster (REC) can manage multiple Redis Enterprise database objects (REDB) in multiple namespaces within the same Kubernetes cluster.
```

Comment:
correctness suggestion:
"""
a single Redis Enterprise operator manages:
  -A single Redis Enterprise Cluster custom resource in the same namespace as the Redis Enterprise operator.
  - Multiple Redis Enterprise Databases custom resources in multiple namespaces that are associated to the same Redis Enterprise Cluster.
"""